frontend/userForm.tsx

```diff
@@ -113,12 +115,17 @@ export function UserForm({id}: {id: number | 'new' | undefined}) {
     protocol(`/api/manage/user/${id == 'new' ? 0 : id}`, postData).then(
       data => {
         switch (data.status) {
-          case "updated": console.info("updated."); break; // updated user
+          case "updated":
+            userMessage('Changes saved', 'success')
```

Comment:
The states such as "success" should be rewritten as enums/consts.

Reply:
Yeah - I should make a plan for legal statuses and messages!

Reply:
`data` is json which has no support for enums/consts, so strings is to be preferred

Reply:
no need to send the enum object